Doc/library/fractions.rst

```diff
@@ -25,8 +25,8 @@ another rational number, or from a string.
 
    The first version requires that *numerator* and *denominator* are instances
    of :class:`numbers.Rational` and returns a new :class:`Fraction` instance
-   with value equal to ``numerator/denominator`` where the denominator is positive.
-   If *denominator* is ``0``, it raises a :exc:`ZeroDivisionError`.
+   with value ``numerator/denominator``. If *denominator* is ``0``, it
```

Comment:
```suggestion
   with value equal to ``numerator/denominator``. If *denominator* is ``0``, it
```
?

Reply:
I would replace "0" with "zero". It raises not only if the denominator is an integer 0.